dav1d-sys/src/lib.rs

```diff
@@ -3,7 +3,664 @@
 #![allow(non_snake_case)]
 #![allow(non_upper_case_globals)]
 
-include!(concat!(env!("OUT_DIR"), "/dav1d.rs"));
+#[repr(C)]
+#[derive(Debug, Copy, Clone)]
+pub struct Dav1dUserData {
+    pub data: *const u8,
+    pub ref_: *mut Dav1dRef,
+}
+
+#[repr(C)]
+#[derive(Debug, Copy, Clone)]
+pub struct Dav1dDataProps {
+    pub timestamp: i64,
+    pub duration: i64,
+    pub offset: i64,
+    pub size: usize,
+    pub user_data: Dav1dUserData,
+}
+
+pub const Dav1dObuType_DAV1D_OBU_SEQ_HDR: Dav1dObuType = 1;
```

Comment:
Stylistic question, but IMHO changing all the enum constants in the following way would reduce noise (they're double-prefixed currently)

```suggestion
pub const Dav1dObuType_SEQ_HDR: Dav1dObuType = 1;
```

Reply:
Actually 

```suggestion
pub const DAV1D_OBU_SEQ_HDR: Dav1dObuType = 1;
```

That way we can even get rid of some style warnings that are silenced currently.